syntax = "proto3";

package SMSC.Protobuf.SMPP;

import "Definition.proto";

message DeliveryReport_Req {
  string smsc_unique_id = 1;

  SMPP_SM smpp = 2;
  Message_Body body = 3;

  string source_client_id = 4;

  string md_message_id = 5;
}

message DeliveryReport_Resp {
  string smsc_unique_id = 1;

  int32 error_code = 2;
}
